app_manager.go

```diff
@@ -20,6 +24,7 @@ func GetAppManager() *AppManager {
 
 func (manager *AppManager) RegistApplication(app interface{}) {
 	manager.applications = append(manager.applications, app)
+	fmt.Printf("Registered app %+v\n", app)
```

Comment:
We should use alog instead of fmt

Reply:
let's not change logger in gofc. Its from open source code. we have logger setup in CNC which redirects stdout/stderr to CNC logs. That's how we could debug blocking issue.